api/handlers/customer_handler.py

```diff
@@ -83,28 +81,26 @@ def insert_customer(self, customer):
                 customer_first_name and customer_last_name and
                 customer_city and latitude and longitude and
                 customer_username and customer_password and
-                customer_phone and customer_cc_type and customer_cc_number and
-                customer_cc_exp and customer_cc_sec_code
+                customer_phone and customer_cc_type and customer_cc_number
             ):
                 location_id = LocationDAO().insert_location(latitude, longitude)
                 user_id = UserDAO().insert_user(
                     customer_username,
                     customer_password,
                     customer_phone
                 )
+                cc_id = CreditCardDAO().insert_credit_card(
+                    customer_cc_type,
+                    customer_cc_number
+                )
                 customer_id = CustomerDAO().insert_customer(
                     customer_first_name,
                     customer_last_name,
                     customer_city,
                     location_id,
+                    cc_id,
```

Comment:
Me percaté que la relacion entre customer y credit card es 1-to-N, y el many side es credit card con total participation. Eso significa que customer no lleva cc_id, sino que credit card lleva el customer_id, hay que invertirlos.